# Data
## Raw data
This data was produced by deviaTE software found at https://github.com/W-L/deviaTE/blob/master/doc/WALKTHROUGH.md
This upload includes 27 different snail lineages and each of the 4 core histone gene sequences.

In total there are 108 files of sequence data.
The "deviate_histones_raw" data is used for the python scipt anyalysis. 

The raw data is set up in a way where each histone gene repeat (H2A for example) was found in the genome and compiled. However, each individual gene cannot be compared. Instead, each nucleotide position of the gene is concatenated and a table is formed. A value of ‘hits’ is given at each nt position for each nucleic acid.This allows a consensus sequence for the histone gene to be formed. However, there may be histone genes that vary at a certain position, or polymorphisms among the repeat histone genes. 

## Lineage estimated copy number and 1x copy number
"Lineage1xAndCN.txt" is used within the python script

Histone abundance was generated by DeviaTE software. This software is designed for estimating the copy number of transposable elements, but is appropriate for histone copy numbers and intragenic sequence variation. It maps reads from whole genome sequence data to one or many repeated sequences and using the coverage (amount of reads mapping to the repeat) as a way to come up a with a measure of genomic abundance or copy number. Copy number is the number of histone copies found within a lineage and can vary for every histone gene. 1x values are the number of hits that occur at each nucleotide position that equal 1 nucleotide( e.g. 1x= 23, there are 92 hits on average per nucleotide at, so there are 4 copies of that gene). 
